tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ng.html

```diff
@@ -54,7 +54,7 @@
   </div>
   <div
     #endFobWrapper
-    *ngIf="timeSelection && timeSelection.end"
+    *ngIf="timeSelection && timeSelection.end && rangeSelectionEnabled"
```

Comment:
What is a scenario in which rangeSelectionEnabled is true and these others are false? I propose we only check rangeSelectionEnabled here.

Reply:
I can't think of a situation where I KNOW this would occur but it seems possible for a render cycle to occur after `rangeSelectionEnabled` is enabled but before `timeSelection.end` is set.

Reply:
That seems like it would be a bug. I am tempted to want that to fail so we know about it. However, it does seem like something we might not catch until it is in production or something and I definitely do not want users running into that...especially as they are trying out a new feature.  Ok I am fine to leave it.